custom/icds_reports/dashboard_utils.py

```diff
@@ -60,3 +61,208 @@ def _get_nav_metadatada():
             'image': static('icds_reports/mobile/images/bulb.png')
         },
     }
+
+
+def _get_nav_menu_items():
+    return {
+        'Maternal and Child Nutrition': [
+            {
+                'name': 'Prevalence of Underweight (Weight-for-Age)',
```

Comment:
nit: it's required on dashboard code, but I think doesn't hurt to mark all these strings for translation. You can do that by just wrapping them like `_('Prevalence of Underweight (Weight-for-Age)')`

Reply:
Do I wrap them like you suggested while defining datastructure in .py file?